docs/weblog/README.md

```diff
@@ -85,21 +85,6 @@ The following text may be written to the body of the response:
 Hello world!\n
 ```
 
-### GET /spans
```

Comment:
Why removing this doc?

Reply:
Ah I think this can stay actually. Initially I had removed this since all the weblog implementations where I tried to use this never actually had implemented it (so it seemed it could be removed) but I do recall seeing at least one valid implementation so I'll undo this